schemas/com.snowplowanalytics.snowplow.media/session/jsonschema/1-0-0

```diff
@@ -0,0 +1,138 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a context entity for media player events that tracks a session of a single media player usage",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow.media",
+    "name": "session",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "mediaSessionId": {
+      "type": "string",
+      "maxLength": 256,
+      "description": "An identifier for the media session."
```

Comment:
We should describe if / when this is reset (or does that description belong in the tracker?)

Reply:
It is not automatically reset, but provided by the user for the media content played in the media player. I have clarified it here and also in the trackers.